<launch>
	<arg name="rviz" default="false" />

	<node type="apc16delft_calibration" pkg="apc16delft_calibration" name="calibration" output="screen">
		<param name="fixed_frame"       value="world"                 />
		<param name="robot_frame"       value="robot_tool0"           />
		<param name="attached_frame"    value="gripper_camera_depth"  />
		<param name="camera_moving"     value="true"                  />
		<param name="camera_namespace"  value="/gripper_camera"       />

		<param name="samples"       value="60"   />
		<param name="radius_min"    value="0.65"  />
		<param name="radius_max"    value="0.75"  />
		<param name="radius_steps"  value="3"    />
		<param name="dome_height"   value="0.075"    />
	</node>

	<!--
	<include file="$(find apc16delft_simulation)/launch/simulator.launch" />

	<include file="$(find apc16delft_camera)/launch/camera.launch">
		<arg name="fake_camera" value="false" />
	</include>
-->
	<node if="$(arg rviz)" type="rviz" pkg="rviz" name="rviz" args="-d '$(find apc16delft_calibration)/config.rviz'" />
</launch>
<!-- vim: set ft=xml: -->
